src/renderer/deviceMessages.ts

```diff
@@ -17,17 +16,6 @@ export async function updateDeviceChats(accountId: number) {
 🖼️ Global Gallery with your pictures, documents, media across all chats
 ✨ many more improvements and bugfixes
 
-Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_41_0
-
-Thanks for testing this pre-release version, for more information and to five us feedback go to https://support.delta.chat/t/help-testing-the-upcoming-1-41-x-release/2793`
+Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_41_0`
```

Comment:
```suggestion
Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_42_0`
```

This needs to be 1.42.0, right?